fcrepo_verify/verifier.py

```diff
@@ -0,0 +1,154 @@
+                else:
+                    logger.warn(
+                        "Resource not in path specified in config file.")
+                    sys.exit(1)
+
+                # skip binaries and fcr:metadata if no binaries exported
+                if not config.bin:
+                    if original.type == "binary" or \
+                            original.origpath.endswith("/fcr:metadata"):
+                        continue
+
+                if filepath.startswith(config.repo):
+                    destination = LocalResource(original.destpath,
+                                                config,
+                                                loggers.file_only)
+                elif filepath.startswith(config.dir):
+                    destination = FedoraResource(original.destpath,
+                                                 config,
+                                                 loggers.file_only)
+
+                if original.type == "binary":
+                    if destination.origpath.endswith(EXT_BINARY_EXTERNAL):
+                        verified = False
+                        verification = "external resource"
+                    if original.sha1 == destination.sha1:
+                        verified = True
+                        verification = original.sha1
+                    else:
+                        verified = False
+                        verification = "{0} != {1}".format(
+                            original.sha1, destination.sha1
+                            )
+
+                elif original.type == "rdf":
+                    if isomorphic(original.graph, destination.graph):
+                        verified = True
+                        verification = \
+                            "{0} triples".format(len(original.graph))
+                    else:
+                        verified = False
+                        verification = ("{0}+{1} triples - mismatch".format(
+                                            len(original.graph),
+                                            len(destination.graph)
+                                            ))
+
+                logger.info("RESOURCE {0}: {1} {2}".format(
+                      total_count(), original.location, original.type
+                      ))
+
+                if not verified:
+                    logger.warn("Resource Mismatch \"{}\"".format(
+                        original.relpath))
+                    failure_count += 1
+                else:
+                    success_count += 1
+
+                if config.verbose:
+                    logger.info("  rel  => {}".format(original.relpath))
+                    logger.info("  orig => {}".format(original.origpath))
+                    logger.info("  dest => {}".format(original.destpath))
+                    logger.info(
+                            "  Verifying original to copy... {0} -- {1}"
+                            .format(verified, verification))
```

Comment:
my preference is for continuation lines to increase indentation level, in line with PEP8